package.json

```diff
@@ -38,7 +38,7 @@
     "chalk": "^2.4.2",
     "conventional-changelog": "3.1.21",
     "conventional-changelog-config-spec": "2.1.0",
-    "conventional-changelog-conventionalcommits": "4.3.0",
+    "conventional-changelog-conventionalcommits": "4.3.1",
```

Comment:
While we're at it, could these all be changed to `^` or at least `~` versions so that we don't have to wait for `conventional-changelog` releases to get simple patch fixes in our own repos?

Reply:
I would agree here, but I didn't want to make this pull request about a practice change, only a dependency update.

Seeing as this is addressed by #650 to a more up-to-date version, I closed this PR. However, I would encourage you to make an issue discussing the suggestion you've made here.